.github/workflows/test-warehouse.yml

```diff
@@ -82,7 +82,7 @@ jobs:
         run: sudo apt-get install python-dev libsasl2-dev gcc
 
       - name: Install dbt
-        run: pip install --pre
+        run: pip install
```

Comment:
Removing the --pre for now because some of the adapter pre-releases are currently broken, and I want to test on the latest stable version